framework/include/media/BufferOutputDataSource.h

```diff
@@ -109,21 +99,15 @@ class BufferOutputDataSource : public OutputDataSource
 	 * @since TizenRT v2.0 PRE
 	 */
 	bool close() override;
-
 	/**
-	 * @brief Puts the file data
-	 * @details @b #include <media/BufferOutputDataSource.h>
-	 * @param[in] buf The buf that buffer to be written to the file
-	 * @param[in] size The size that the size of the buffer
-	 * @return if there is nothing to write, it returns 0
-	 *         if error occurred, it returns -1, else written size returns
-	 * @since TizenRT v2.0 PRE
+	 * @brief Called when output stream buffer is readable
+	 * @details @b #include <media/FileOutputDataSource.h>
+	 * @return size of data read from the stream buffer, -1 means error occurred.
+	 * @since TizenRT v2.0
```

Comment:
Currently PRE is correct.

Reply:
BufferOutputDataSource doesn't need to use stream buffer?
To use stream buffer, `write` method is implemented in OutputDataSource class.

Reply:
He means the TizenRT version. not v2.0 but v2.0 PRE. Please check the comment once more.

Reply:
Fixed